src/mumble/AudioConfigDialog.cpp

```diff
@@ -640,8 +640,10 @@ AudioOutputDialog::AudioOutputDialog(Settings &st) : ConfigWidget(st) {
 	qcbSystem->setEnabled(qcbSystem->count() > 1);
 
 	qcbLoopback->addItem(tr("None"), Settings::None);
-	qcbLoopback->addItem(tr("Local"), Settings::Local);
-	qcbLoopback->addItem(tr("Server"), Settings::Server);
+	qcbLoopback->addItem(tr("Local (don't send to others)"), Settings::LocalOnly);
+	qcbLoopback->addItem(tr("Local (send to others)"), Settings::LocalRegular);
+	qcbLoopback->addItem(tr("Server (don't send to others)"), Settings::ServerOnly);
+	qcbLoopback->addItem(tr("Server (send to others)"), Settings::ServerRegular);
```

Comment:
```suggestion
	qcbLoopback->addItem(tr("Local only"), Settings::LocalOnly);
	qcbLoopback->addItem(tr("Local + Broadcast"), Settings::LocalRegular);
	qcbLoopback->addItem(tr("Server only"), Settings::ServerOnly);
	qcbLoopback->addItem(tr("Server + Broadcast"), Settings::ServerRegular);
```
I feel like my suggestion can still be improved upon :thinking: 

Either way, I think we need tooltips for all of these that explain in more detail what is meant by that

Reply:
Yes, absolutely.